src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldMedia.class.php

```diff
@@ -73,12 +74,29 @@ protected function getHtmlRenderedHtml($bReadOnly)
 
             $iWidth = 0;
             $iHeight = 0;
-            if (isset($oImage->aData) && isset($oImage->aData['height'])) {
-                $iHeight = $oImage->aData['height'];
-            }
-            if (isset($oImage->aData) && isset($oImage->aData['width'])) {
-                $iWidth = $oImage->aData['width'];
+            if (isset($oImage->aData)) {
+                if (isset($oImage->aData['height'])) {
+                    $iHeight = $oImage->aData['height'];
+                } else {
+                    if (isset($this->oTableRow->sqlData['width'])) {
+                        $oImage->aData['height'] = $this->oTableRow->sqlData['height'];
+                        $iHeight = $oImage->aData['height'];
+                    } else {
+                        $oImage->aData['height'] = $iHeight;
```

Comment:
This is still 0?

Then 0 instead of $iHeight should be used.

Reply:
done